exercises/crypto-square/crypto_square_test.py

```diff
@@ -46,7 +47,7 @@ def test_long_string(self):
         msg = ('If man was meant to stay on the ground, god would have given '
                'us roots.')
         ciph = 'imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn sseoau'
-        self.assertEqual(ciph, encode(msg))
+        self.assertEqual(encode(msg), ciph)
```

Comment:
Track-specific tests `test_small_imperfect_square`, `test_punctuation_and_numbers`, and `test_long_string` should be removed because they conflict canonical tests.

Reply:
I removed they